website/templates/util/render_node.mako

```diff
@@ -51,16 +51,30 @@
             % endif
             </span>
 
-            <!-- Show/Hide recent activity log -->
-            % if not summary['archiving']:
             <div class="pull-right">
                 % if not summary['primary'] and 'write' in user['permissions'] and not node['is_registration']:
                     <i class="fa fa-times remove-pointer" data-id="${summary['id']}" data-toggle="tooltip" title="Remove link"></i>
                     <i class="fa fa-code-fork" onclick="NodeActions.forkPointer('${summary['id']}', '${summary['primary_id']}');" data-toggle="tooltip" title="Fork this ${summary['node_type']} into ${node['node_type']} ${node['title']}"></i>
                 % endif
-                <i id="icon-${summary['id']}" class="pointer fa fa-angle-down" onclick="NodeActions.openCloseNode('${summary['id']}');" style="font-weight:bold;"></i>
+                % if summary['logged_in'] and summary['is_contributor']:
```

Comment:
Node links also get this button at the moment. My guess is that should be restricted. You may need to check with product on how they want node links handled. 
<img width="526" alt="screen shot 2017-05-26 at 10 00 18 am" src="https://cloud.githubusercontent.com/assets/1322421/26497649/337f0712-41fa-11e7-89d1-601f370ddd82.png">